CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 # Export libraries along with our DLLs if we want to build shared
 # Allow the developer to select if Dynamic or Static libraries are built
 
-include (GenerateExportHeader)

```

Comment:
I'd leave this line in - it's still performing useful checks that we might want to keep.